app/controllers/FileNotFoundController.scala

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import java.util.UUID
+
+import auth.TokenSecurity
+import configuration.KeycloakConfiguration
+import javax.inject.Inject
+import org.pac4j.play.scala.SecurityComponents
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request}
+
+class FileNotFoundController @Inject()(val controllerComponents: SecurityComponents,
```

Comment:
I've added notes to the remaining subtasks to remind us to remove these controllers and routes once we've connected the error pages to the Play error handling. 👍

Reply:
Great, thanks! :)